### PR TITLE
[4.0.0] [Choreo Connect] Performance test result document addition

### PR DESCRIPTION
**Pull request by slahirucd7**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -317,6 +317,10 @@ nav:
                 - File Upload via Multipart Requests: deploy-and-publish/deploy-on-gateway/choreo-connect/file-upload-via-multipart-requests.md
                 - Token Caching: deploy-and-publish/deploy-on-gateway/choreo-connect/configure-caching.md
                 - Production Deployment Guideline: deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md
+                - Performance Tests: 
+                    - Performance Test Overview: deploy-and-publish/deploy-on-gateway/choreo-connect/performance-tests/performance-test-results-overview-cc.md
```
Comment (pubudu538) [inline en/mkdocs.yml:321]: Let's call this Overview

**Review by pubudu538 — APPROVED** [review]

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/performance-tests/router-with-one-cpu-for-rest-api.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+These test results were obtained considering a Choreo Connect deployment with default values. Therefore the concurrency level used for the tests was two. Below table shows resource allocations for the Choreo Connect components.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/performance-tests/router-with-one-cpu-for-rest-api.md:1]: Let's add a proper title for this page.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/performance-tests/performance-test-results-overview-cc.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+# Performance Test Results
+
+The performance of Choreo Connect was measured using an API that invoke a simple "Netty Echo Service". The task covered by the Netty service is to echo back any request posted to the service. The API used for the test cases was a secured API, which directly invokes the backend through Choreo Connect using JWT tokens.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/performance-tests/performance-test-results-overview-cc.md:3]:
```suggestion
The performance of Choreo Connect was measured using an API that invokes a simple "Netty Echo Service". The task covered by the Netty service is to echo back any request posted to the service. The API used for the test cases was a secured API, which directly invokes the backend through Choreo Connect using JWT tokens.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/performance-tests/router-with-one-cpu-for-rest-api.md (hunk 2 above)
Comment (slahirucd7) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/performance-tests/router-with-one-cpu-for-rest-api.md:3]: fixed with: https://github.com/wso2/docs-apim/pull/6099/commits/f93dd8263b73034b73bc8e85c773c059c0bed41e

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**